techeria_controller/authentication/migrations/0001_initial.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 3.2.7 on 2021-11-24 07:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BuyerModel',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('first_name', models.CharField(max_length=255)),
+                ('last_name', models.CharField(max_length=255)),
+                ('user_name', models.CharField(max_length=255)),
+                ('date_of_birth', models.DateField()),
+                ('email', models.EmailField(max_length=254)),
+                ('mobile_number', models.IntegerField()),
```

Comment:
It would be great to have a  PhoneNumberField.  How about setting up the field in a way that it can store international phone number as well